Resources/Prototypes/tags.yml

```diff
@@ -67,6 +67,9 @@
 - type: Tag
   id: Bloodpack
 
+- type: Tag
+  id: BloodTomato
+
```

Comment:
This should be placed in Resources/Prototypes/_Floof/tags.yml (namespace change)